docker-compose.dev.yml

```diff
@@ -32,13 +33,20 @@ services:
     command: /bin/bash -c 'python -m venv .backend_env && source .backend_env/bin/activate && pip install -U pip setuptools wheel && pip install -r requirements.txt && python -m ibutsu_server --host 0.0.0.0'
     working_dir: /mnt
     volumes:
-      - "./backend:/mnt"
+      - "./backend:/mnt:z"
     user: ${UID:-1000}
     ports:
       - "8080:8080"
     depends_on:
-      - postgres
-      - redis
+      postgres:
+        condition: service_healthy
+      redis:
+        condition: service_started
+    healthcheck:
+      test: curl --output /dev/null --silent --head --fail http://localhost:8080
```

Comment:
This should probably be something like `/api/health` -- there is a health endpoint.

Reply:
Fixed